### ISSUE TITLE
pre-trained models

### ISSUE THREAD

**Issue by yyash-patel** [jskvrna/MonoSOWA#2]
Hi,

Great work !

I trained the MonoDETR model on kitti labels generated using MonoSOWA, but I am not able to match the results shown in table VI (first column- KITTI) in the paper.

I was hoping if you will be releasing MonoDETR model trained on Kitti pseudo labels?

Thanks !

**Comment by seokyeong94** [comment]
Hi, I’m also having trouble reproducing the results on KITTI.
Could you kindly share a link to the generated pseudo-labels or the pretrained model weights?

**Comment by jskvrna** [comment]
Dear all,

Sorry for the late response.

Please find the pertained checkpoints under MonoDETR/checkpoints. For the kitti one please use the best_kitti one. We will add the corresponding config files for each checkpoint as the config has to match the pertained weights. 

It is worth noting that the training of the MonoDETR is quite unstable, so to achieve the best performance you need to both train the model multiple times and cherry pick the best checkpoint. Usually the last checkpoint is not the best.

Also for the pseudo-labels generation it depends also much on the config you are using. You can share yours config files and I can guide you how to achieve the best results.

Best,
Jan

**Comment by seokyeong94** [comment]
Thank you very much for your work. 
However, I think there's something wrong with the provided checkpoints. If we run it, the AP3D become zero for all cases while AP2D scores are reasonable. 
And do you have any plan to release the pretrained model weights that can used to validate on kitti-360 and waymo?

**Comment by jskvrna** [comment]
Hi,

That sounds like a problem with a incorrectly set focal length. Due to MonoDETR depth bin prediction, when working with the KITTI dataset only we do not use the canonical object space. Try to turn it off in the config and it should work as expected for the only KITTI model.

You can find the checkpoints for KITTI-360 in the same folder. Due to license reasons it is not possible to publish weights trained on Waymo dataset. 

Best,
Jan
